uw_oidc/middleware.py

```diff
@@ -45,8 +46,9 @@ def process_view(self, request, view_func, view_args, view_kwargs):
                         request.session[self.TOKEN_SESSION_KEY] = auth_token
 
             except InvalidTokenError as ex:
-                return HttpResponse(status=401,
-                                    reason='Invalid token: {}'.format(ex))
+                return HttpResponse(
+                    status=int(getattr(settings, "TOKEN_ERR_CODE", 401)),
```

Comment:
This should not be configurable in settings, 401 must be the only status code returned for InvalidTokenError.